### PR TITLE
Added custom toolbar sample in React

### PR DESCRIPTION
**Pull request by Gayathri-github7**
Hi Team,

Added custom toolbar sample in React. Kindly review the changes and provide your suggestion.

Regards,
Gayathri R

### CODE REVIEW

File: Custom Toolbar/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import {registerLicense} from '@syncfusion/ej2-base';
+registerLicense("Mgo+DSMBMAY9C3t2V1hhQlJAfV5AQmBIYVp/TGpJfl96cVxMZVVBJAtUQF1hSn5Vd0BiXX9ccXRQRGBa")
```
Comment (SasidharanKaruppiah) [inline Custom Toolbar/src/index.tsx:7]: is this trial license key ?

**Review by SasidharanKaruppiah — CHANGES_REQUESTED** [review]

File: Custom Toolbar/src/index.tsx (hunk 1 above)
Comment (Gayathri-github7) [reply Custom Toolbar/src/index.tsx:7]: yes its trial license key.

File: Custom Toolbar/src/react-app-env.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```
Comment (rparthi-pdf) [inline Custom Toolbar/src/react-app-env.d.ts:1]: Do we need this empty file?

File: Custom Toolbar/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,769 @@
...
+  }
+
+  function onPageChanged() {
+    let currentPageNumber = viewer.currentPageNumber;
+    let inputElement: HTMLInputElement = document.getElementById('currentPage') as HTMLInputElement;
+    inputElement.value = currentPageNumber.toString();
+  }
+
+  function onDocumentLoaded() {
+    let pageCount = viewer.pageCount;
```
Comment (rparthi-pdf) [inline Custom Toolbar/src/App.tsx:389]: Can we specify the type for this variable? Additionally, please review the code changes for the same scenario and update the variable with the appropriate type.

**Review by rparthi-pdf — CHANGES_REQUESTED** [review]

File: Custom Toolbar/src/react-app-env.d.ts (hunk 2 above)
Comment (Gayathri-github7) [reply Custom Toolbar/src/react-app-env.d.ts:1]: This file is generated automatically during project creation. Should it be removed?

File: Custom Toolbar/src/App.tsx (hunk 3 above)
Comment (Gayathri-github7) [reply Custom Toolbar/src/App.tsx]: I have modified the changes as per your suggestion.

File: Custom Toolbar/src/react-app-env.d.ts (hunk 2 above)
Comment (rparthi-pdf) [reply Custom Toolbar/src/react-app-env.d.ts:1]: If it is a default file, then we can ignore it.

**Review by rparthi-pdf — APPROVED** [review]

**Pull request merged**